### ISSUE TITLE
Build scripts Again

### ISSUE THREAD

**Issue by bitcreditscc** [bitcoin/bitcoin#6406]
I think someone modified the build scripts again, pulled main branch and tried to build on windows

./configure: line 23605: syntax error near unexpected token `QTPLATFORM,'
./configure: line 23605: `        PKG_CHECK_MODULES(QTPLATFORM, Qt5PlatformSupport, QT_LIB
S="$QTPLATFORM_LIBS $QT_LIBS")'

**Comment by laanwj** [comment]
I don't understand your issue title nor the description.

The error seems to indicate that you're missing some m4 file or part of autotools.

**Comment by theuni** [comment]
This could be a problem on windows if pkg-config isn't installed (it's not required there). I'll have a look.

**Comment by theuni** [comment]
Blehk, I see what's going on. pkg-config is required for windows, even though we don't use it. This could be fixed up and guarded, but since we've been requiring it accidentally for this long, it may make more sense to just go ahead and use it.

tl;dr: install pkg-config, that should fix your problem. For the future, we may look into actually using it for Windows.

**Comment by Bushstar** [comment]
Posting in case this helps anyone, this occurs when compiling the 0.11 branch on Windows. Installed pkg-config binaries, resolved its few missing DLLs but still had the same problem. pkg-config-lite solved it for me.

[http://sourceforge.net/projects/pkgconfiglite/files/0.28-1/pkg-config-lite-0.28-1.tar.gz/download](http://sourceforge.net/projects/pkgconfiglite/files/0.28-1/pkg-config-lite-0.28-1.tar.gz/download)

Configure, compile and install with the following then run autogen.sh.

    configure --prefix=/usr && make && make install
